applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAXNativeAdView.java

```diff
@@ -423,22 +424,29 @@ private void addMediaView(final MethodCall call)
 
         if ( mediaViewContainer == null )
         {
-            mediaViewContainer = new FrameLayout( context );
+            mediaViewContainer = new RelativeLayout( context );
             // Sets an identifier for the Google adapters to verify the view in the tree
             mediaViewContainer.setId( MEDIA_VIEW_CONTAINER_TAG );
             mediaViewContainer.setTag( MEDIA_VIEW_CONTAINER_TAG );
             nativeAdView.addView( mediaViewContainer );
         }
 
+        Rect rect = getRect( call );
+
         if ( mediaView.getParent() == null )
         {
-            mediaViewContainer.addView( mediaView );
-
-            mediaView.getLayoutParams().height = ViewGroup.LayoutParams.MATCH_PARENT;
-            mediaView.getLayoutParams().width = ViewGroup.LayoutParams.MATCH_PARENT;
+            RelativeLayout.LayoutParams params = new RelativeLayout.LayoutParams( rect.width(), rect.height() );
+            params.addRule( RelativeLayout.ALIGN_PARENT_TOP, RelativeLayout.TRUE );
+            params.addRule( RelativeLayout.ALIGN_PARENT_LEFT, RelativeLayout.TRUE );
```

Comment:
If the goal is to make the child fit the parent, you should use:

```
RelativeLayout.LayoutParams layoutParams = new RelativeLayout.LayoutParams(
    RelativeLayout.LayoutParams.MATCH_PARENT, // Width matches the parent
    RelativeLayout.LayoutParams.MATCH_PARENT  // Height matches the parent
);
```

Reply:
Thanks, updated `RelativeLayout.LayoutParams` with `RelativeLayout.LayoutParams.MATCH_PARENT` and tested again with all the adapters.